commit 5d0ffa2b84e6f0fdc5bf96c0de6178f3d2779544
Author: Don Mullis <dwm@meer.net>
Date:   Fri Dec 8 02:39:50 2006 -0800

    [PATCH] fault-injection: Correct, disambiguate, and reformat documentation
    
    Correct, disambiguate, and reformat documentation.
    
    Signed-off-by: Don Mullis <dwm@meer.net>
    Cc: Akinobu Mita <akinobu.mita@gmail.com>
    Signed-off-by: Andrew Morton <akpm@osdl.org>
    Signed-off-by: Linus Torvalds <torvalds@osdl.org>

diff --git a/Documentation/fault-injection/failmodule.sh b/Documentation/fault-injection/failmodule.sh
index 9abac34633b9..474a8b971f9c 100644
--- a/Documentation/fault-injection/failmodule.sh
+++ b/Documentation/fault-injection/failmodule.sh
@@ -26,6 +26,6 @@ fi
 # Disable any fault injection
 echo 0 > /debug/$1/stacktrace-depth
 
-echo `cat /sys/module/$2/sections/.text` > /debug/$1/address-start
-echo `cat /sys/module/$2/sections/.exit.text` > /debug/$1/address-end
+echo `cat /sys/module/$2/sections/.text` > /debug/$1/require-start
+echo `cat /sys/module/$2/sections/.exit.text` > /debug/$1/require-end
 echo $STACKTRACE_DEPTH > /debug/$1/stacktrace-depth